tests/unit/django_spanner/test_base.py

```diff
@@ -0,0 +1,108 @@
+# Copyright 2020 Google LLC
+#
+# Use of this source code is governed by a BSD-style
+# license that can be found in the LICENSE file or at
+# https://developers.google.com/open-source/licenses/bsd
+
+import sys
+import unittest
+
+from mock_import import mock_import
+from unittest import mock
+
+
+@mock_import()
+@unittest.skipIf(sys.version_info < (3, 6), reason="Skipping Python 3.5")
```

Comment:
I think we can safely drop 3.5 in tests since we already decided not to support it elsewhere.